### PR TITLE
[QA] US-Update-Gem-For-Our-Account

### PR DESCRIPTION
**Pull request by timsegraves**
This is a big update to the zuora gem so it supports custom fields and it has the updated WSDLs as well.  I hadn't made these changes to a branch originally since it was just in my repo.

### CODE REVIEW

File: lib/zuora/soap_connector.rb [hunk 1]
```diff
@@ -32,7 +40,14 @@ def update
...
           obj_attrs.reject{|k,v| @model.read_only_attributes.include?(k) }.each do |k,v|
-            a.__send__(ons, k.to_s.camelize.to_sym, v) if change_syms.include?(k)
+            key = k
+            if @model.custom_attributes.include?(k)
+              key = k.to_s[0..-4].camelize + '__c'
+              key = key.to_sym
+            else
+              key = k.to_s.camelize.to_sym
+            end
+            a.__send__(ons, key, v) if change_syms.include?(k)
```
Comment (ben-grid) [inline lib/zuora/soap_connector.rb:50]: Should we be checking if change_syms includes k, or key?
i.e.
a.**send**(ons, key, v) if change_syms.include?(key)

**Comment by ben-grid** [comment]
CR done, one question.

File: lib/zuora/soap_connector.rb (hunk 1 above)
Comment (timsegraves) [inline lib/zuora/soap_connector.rb:50]: Nah, in this case key is the converted value and k is value from the model prior to conversion to camelcase.  The models define the fields with underscores (i.e. account_id) and they are then converted to camelcase for the actual SOAP call.  

It's basically the same code as before I just added the if block in the middle to account for custom fields (__c).

**Comment by timsegraves** [comment]
Ready for QA

**Comment by timsegraves** [comment]
Closing this PR and opening a new one with a new branch.

**Pull request closed without merging**